src/asynk/async_worker_pool.rs

```diff
@@ -1,7 +1,7 @@
 use crate::asynk::async_queue::AsyncQueueable;
 use crate::asynk::async_queue::DEFAULT_TASK_TYPE;
 use crate::asynk::async_worker::AsyncWorker;
-use crate::asynk::AsyncError as Error;
+use crate::FangError as Error;
```

Comment:
why do we need to alias? (the same comment for other placnes)

Reply:
[elete aliases](https://github.com/ayrat555/fang/pull/77/commits/f9b9ae464327116747ad62de38c61ce0b4279376)